bin/gistup

```diff
@@ -175,8 +175,11 @@ function createGist(token, callback) {
   });
 }
 
-function gitRemoteAdd(id, callback) {
-  child.exec("git remote add --track master " + quote.single(argv.remote) + " git@gist.github.com:" + id + ".git", function(error, stdout, stderr) {
+function gitRemoteAdd(id, settings, callback) {
+  url = "git@gist.github.com:" + id + ".git"; // default
+  if (/^https$/i.test(settings.protocol)) url = "https://" + settings.user + ":" + settings.token + "@gist.github.com/" + id + ".git";
```

Comment:
settings.user is not needed, you can simply construct the URL as: https://TOKEN@gist.github.com